packages/perseus/src/widgets/radio/option-status.jsx

```diff
@@ -23,61 +23,65 @@ class OptionStatus extends React.Component<{
     previouslyAnswered: boolean,
     reviewMode: boolean,
     satStyling?: boolean,
-    ...
-}> {
-    _renderText(): string {
-        const {checked, correct, crossedOut} = this.props;
-        if (correct) {
-            // For correct answers, we surface checked _or_ crossedOut state,
-            // because any interaction with the correct answer is noteworthy!
-            if (checked) {
-                return i18n._("Correct (selected)");
-            }
-            if (crossedOut) {
-                return i18n._("Correct (but you crossed it out)");
-            }
-            return i18n._("Correct");
-        }
-        // But, for incorrect answers, we only surface checked state,
-        // because crossing out an incorrect answer is not noteworthy.
+|};
+
+function renderText(checked, correct, crossedOut): string {
```

Comment:
It'd be good to add Flow types to the parameters here.  I'm assuming Flow is correctly inferring them right now, but it's much clearer to be explicit in function definitions.

Reply:
Thank you! Would it be better just to add "boolean" after each parameter or to reference OptionStatusProps?
For example:
`checked: boolean` or `checked: OptionStatusProps["checked"]`

Reply:
Yes. That's what I meant by adding Flow types. :) 
```suggestion
function renderText(checked: boolean, correct: boolean, crossedOut: boolean): string {
```